src/main/java/com/example/GymProject/dao/TrainerDao.java

```diff
@@ -14,11 +16,12 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.logging.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Repository
 public class TrainerDao {
-    private final Logger logger = Logger.getLogger(this.getClass().getName());
+    private final Logger logger = LoggerFactory.getLogger(this.getClass().getName());
     private final Map<String, Trainer> trainerMap = new HashMap<>();
     @Value("${trainerFilePath}")
```

Comment:
This value is not resolved correctly. To resolve it, please add the following annotation to your `AppConfig.java`
`@PropertySource("classpath:application.properties")`